views/page-widgets/get-pro.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Widget view.
+ *
+ * @package Progress_Planner
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+?>
+
+<a class="get-pro-link" href="https://progressplanner.com/checkout/?edd_action=straight_to_gateway&download_id=1136" target="_blank">
```

Comment:
We should make this a redirect / shortlink so that when we have to change anything we can...